packages/client/src/api/client.ts

```diff
@@ -274,6 +274,18 @@ class DatabaseApi {
     });
   }
 
+  public patchDatabaseMetadata(
```

Comment:
What about `updateDatabaseMetadata`? I know the openapi spec id uses `patch` in its name, but... we are not doing `patch()` when updating records for example.

Reply:
Agreed I even had a comment on the openapi side but I was ignored 😅